### DISCUSSION TITLE
Freeze on AVI import

### DISCUSSION THREAD

**Discussion by vovodroid** [audacity/audacity#3262] [Q&A] [answered]
**Describe the bug**
Starting from v 3.1 Audacity freeze when drag and drop mp4 files in main window (FFMpeg installed). Latest working version is 3.0.5.

1. Go to folder with several mp4 files
2. Drag and drop them in main window
3. Audacity starts to export and next freezes.

 - OS:  Windows 10 x64
 - Version >= 3.1 x64

**Comment by LWinterberg** [comment]
I could not reproduce this with the 20 or so random files I've got in my download folder. They import just fine. 

* Do you have some more information on the nature of your files? 
    * Are they all coming out of a specific program? 
    * Also, can you run one of the files through [mediainfo](https://mediaarea.net/en/MediaInfo) and paste the text output here?
* What FFMPEG version are you using? You can check in Audacity's Preferences > Libraries

**Comment by vovodroid** [comment]
Here is FFMPeg version: https://pasteboard.co/qY3f1LboLsrX.png

Also sample files:
[20220124_000253.zip](https://github.com/audacity/audacity/files/7924824/20220124_000253.zip)

As I mentioned, I reverted to version 3.0.5 to get import working. In v 3.1 problem is especially when I drag and drop list of file.

**Comment by LWinterberg** [comment]
Okay. Those are AVIs rather than MP4s, and I can reproduce a short freeze (2s) when dragging and dropping a group of files into the window.

**Comment by vovodroid** [comment]
Sorry, indeed AVI, I just confused extension.

On my system it's permanent freeze with rectangle "Move 20 files" in the screen, even after tracks were appeared.
I'm ready to reproduce it with logs on, specific build - whatever needed.

**Comment by LWinterberg** [comment, marked as answer]
Does this also happen with more recent versions of FFMPEG? The latest version that 3.1.3 supports is 4.4/avformat-58. You can download it from eg https://www.gyan.dev/ffmpeg/builds/packages/ffmpeg-4.4.1-full_build-shared.7z

**Reply by vovodroid** [reply]
Well, it helps, thanks!

But button "Download" FFmpeg in version 3.1.3

![image](https://user-images.githubusercontent.com/8691781/180495982-96b4162d-5251-467f-b6dd-6880d9a97ca2.png)

still leads to old version of FFmpeg with the link chain:

file:///C:/Program%20Files/audacity/help/manual/man/faq_installing_the_ffmpeg_import_export_library.html
file:///C:/Program%20Files/audacity/help/manual/man/installing_and_updating_audacity_on_windows.html#winff
file:///C:/Program%20Files/audacity/help/manual/man/installing_ffmpeg_for_windows.html
https://lame.buanzo.org/ffmpeg64audacity.php
https://lame.buanzo.org/FFmpeg_v2.2.2_for_Audacity_on_Windows_64bit.exe

**Reply by vovodroid** [reply]
May be open bug about outdated help files?
